bitherj/src/main/java/net/bither/bitherj/core/PeerManager.java

```diff
@@ -768,14 +768,14 @@ public void run() {
                                         .getLastBlock().getTxHashes().size());
                         AbstractApp.notificationService.sendLastBlockChange();
                     } else {
-                        abandonPeer(fromPeer);
-                        log.info("Peer {} relay {}/{} block. drop this peer",
+                        // abandonPeer(fromPeer); // peer should not be abandoned because of orphan block
```

Comment:
no need to keep inside comment.
also, if logging orphan block hash is tricky you can log a line like this:
`log.info("Peer {} relay {}/{} block. Possible orphan block encountered, NOT dropping peer",`